sampul/konten-en.tex

```diff
@@ -1,32 +1,47 @@
-% Ubah kode buku berikut dengan yang ditentukan oleh departemen dalam Bahasa Inggris
-FINAL PROJECT - TD123456
-
-\vspace{6ex}
-
-% Ubah kalimat berikut dengan judul tugas akhir dalam Bahasa Inggris
+% Ubah kode buku berikut dengan yang ditentukan oleh departemen
 \begin{large}
-  \emph{ANTI-GRAVITY} BASED ENERGY CALCULATION ON OUTER SPACE ROCKETS
+  FINAL PROJECT - \coursecode{}
 \end{large}
 
-\vspace{4ex}
+\vspace{\fill}
+
+% Ubah kalimat berikut dengan judul tugas akhir
+\begin{spacing}{1.5}
+  \begin{Large}
+    \engtatitle
+  \end{Large}
+\end{spacing}
+
+\vspace{\fill}
 
 % Ubah kalimat-kalimat berikut dengan nama dan NRP mahasiswa
-Elon Reeve Musk \\
-NRP 0123 20 4000 0001
+\begin{large}
+  \name{} \\
+  \textmd{NRP \nrp{}}
+\end{large}
 
-\vspace{2ex}
+\vspace{\fill}
 
 % Ubah kalimat-kalimat berikut dengan nama-nama dosen pembimbing
-Advisor \\
-Nikola Tesla, S.T., M.T. \\
-Wernher von Braun, S.T., M.T.
+\begin{large}
+  \textmd{Advisor} \\
+  \advisor{} \\
+  \textmd{NIP \advisornip{}} \\
+  \coadvisor{} \\
+  \textmd{NIP \coadvisornip{}}
+\end{large}
+
+\vspace{\fill}
+
+% Ubah kalimat-kalimat berikut dengan nama departemen dan fakultas
+Undergraduate Study Program of \engstudyprogram{} \\
 
-\vspace{6ex}
+\mdseries
 
-% Ubah kalimat-kalimat berikut dengan nama departemen dan fakultas dalam Bahasa Inggris
-DEPARTMENT OF AEROSPACE ENGINEERING \\
-Faculty of Aerospace Technology \\
+Department of \engdepartment{} \\
+Faculty of \engfaculty{} \\
 Sepuluh Nopember Institute of Technology
 
 % Ubah kalimat berikut dengan tempat dan tahun pembuatan buku
-Surabaya 2021
+\place{} \\
+\the\year{}
```

Comment:
It seems single line is enough, since it's just a variable, won't be edited. Check the other lines too.